nix/development.nix

```diff
@@ -51,6 +51,7 @@
       go
       go2nix
       godep
+      jre
```

Comment:
You shouldn't need this dependency here, unless you actually want it as a system dep. Adding it under openhab2's config should work.